wheatley/tower.py

```diff
@@ -249,7 +249,8 @@ def _join_tower(self) -> None:
 
     def _request_global_state(self) -> None:
         """ Send a request to the server to get the current state of the tower. """
-        self._emit('c_request_global_state', {"tower_id": self.tower_id}, "Request state")
+        self.logger.debug("Requesting global state.")
+        self._emit('c_request_global_state', {"tower_id": self.tower_id}, "")
```

Comment:
It would probably make more sense to leave the logging up to the callers.  For example, some things (like calling `Bob` or `Single`) _should_ be logged at `INFO`, whereas things like requesting global state should probably be `DEBUG` and things like sending the signals to ring bells should not be logged.

I'll fix this now.